main.go

```diff
@@ -129,6 +140,28 @@ func sendMetrics(event *corev2.Event) error {
 		return err
 	}
 
+	// Add the check status field as a metric if requested. Measurement recorded as the check name.
+	if config.CheckStatusMetric {
+		var tagList = make([]*corev2.MetricTag, 0)
+		tagList = append(tagList,
+			&corev2.MetricTag{
+				Name:  "namespace",
+				Value: event.Entity.Namespace,
+			})
```

Comment:
We probably want to make sure the tags are consistent across all metrics that get passed through this handler. We can combine this tag logic with the tag logic you see starting at L166 into a separate function to reuse the same tags!

Reply:
Yeah, I added the namespace tag in addition to the sensu_entity_name on L185 because I needed to bootstrap the event with metric data since a normal check has no concept of "tags". Consistency is a good idea though so I think it may be easier just to drop the namespace tag in this PR for now. Populate an empty slice, and use the existing logic until something like [issue-2160](https://github.com/sensu/sensu-go/issues/2160) is implemented. Thoughts?